Blog post - Translation DevGuide to Portuguese (#1186)
ropensci/roweb3: content/blog/2025-11-25-translation-devguide-pt/index.md

Below is a list of the contributors, in alphabetical order:

-   António Pedro, Angola

-   Ariana Moura Cabral, Brazil

-   Beatriz Milz, Brazil

-   Daniel Vartanian, Brazil

-   Francesca Belem Lopes Palmeira, Brazil

-   Ildeberto Vasconcelos, Angola

-   João Granja-Correia, Portugal

-   Marcelo Perlin, Brazil

-   Pedro Faria, Brazil

-   Rafael Fontenelle, Brazil

-   Samuel Carleial Fernandes, Brazil

In addition, the translation project was coordinated by [Yanina Bellini Saibene](https://yabellini.netlify.app/), rOpenSci’s Community Manager. Other members of the rOpenSci team also played key roles in reviewing Pull Requests, including [Maëlle Salmon](https://masalmon.eu/).

> ## Testimonials

Author: I think your suggestion is good! Thanks.

One interesting aspect of this project was the methodology used, which relied on tools such as [Quarto](https://quarto.org/), [`{rmarkdown}`](https://rmarkdown.rstudio.com/), [`{babeldown}`](https://docs.ropensci.org/babeldown/index.html), and [DeepL](https://developers.deepl.com/docs/getting-started/intro). This approach proved highly effective in several steps of the translation process, and it may be a compelling option for others seeking a methodology that: 1) is open and inclusive for its audience; 2) introduces some degree of automation into the workflow; and 3) provides good file organization for the project (since, depending on the size of the material, file management can become a crucial part of the workflow).
{{< /quote >}}

{{< quote footer="Ariana Moura Cabral">}}

> For me, contributing to this translation project was both a learning journey and an exchange of experiences. On one hand, I was able to meet people who, like me, value knowledge sharing and believe in the transformative power of words, learn more about rOpenSci’s values, and update myself on best practices for developing and managing R packages. On the other hand, I had the chance to contribute, even if just a little, to someone else's learning, which is deeply rewarding to me. I hope the translated material opens new pathways for those who also appreciate the R language and wish to explore its vast universe. In the end, I see this experience as a continuous cycle of sharing and growth, which reminds me of the words of Cora Coralina: “Happy is he who transfers what he knows and learns what he teaches.”

Reviewer: ```suggestion
For me, contributing to this translation project was both a learning journey and an exchange of experiences. On one hand, I was able to meet people who, like me, value knowledge sharing and believe in the transformative power of words, learn more about rOpenSci’s values, and update myself on best practices for developing and managing R packages. On the other hand, I had the chance to contribute, even if just a little, to someone else's learning, which is deeply rewarding to me. I hope the translated material opens new pathways for those who also appreciate the R language and wish to explore its vast universe. In the end, I see this experience as a continuous cycle of sharing and growth, which reminds me of the words of Cora Coralina: "Happy are those who share what they know and learn as they teach".
```

During the review process, particular attention was also given to reducing gendered language (for example, changing “os usuários” (the users) to “os(as) usuários(as)” or “as pessoas usuárias”)[^2], while still prioritizing fluid and inclusive writing. Additionally, because automatic translation tends to translate everything, another recurring concern was restoring proper names (books, packages, or functions) back to English when appropriate.

[^2]: Translation note: In English, the expression “the users” is gender-neutral, but in Portuguese it defaults to the masculine form. In this translation, inclusive alternatives were adopted where appropriate to avoid unnecessary gender marking.

One barrier that remains for readers of the Portuguese Dev Guide is that most of the function names, package documentation, and book references still point to English-language content. However, the rOpenSci team has been working to translate blog posts referenced in the Dev Guide into Portuguese, which gradually helps reduce this gap.

## Impact and Next Steps

> The Portuguese version of the Dev Guide becomes a valuable study resource across Portuguese-speaking countries, not only for those interested in developing R packages following best practices, but also for anyone seeking to better understand rOpenSci’s submission guidelines and peer review process. It helps open doors, reduce language barriers, and strengthen open science.

Reviewer 2: ```suggestion
The Portuguese version of the Dev Guide is a valuable study resource across Portuguese-speaking communities, not only for those interested in developing R packages following best practices, but also for anyone seeking to better understand rOpenSci’s submission guidelines and peer review process. It helps open doors, reduce language barriers, and strengthen open science.
```

---
slug: translation-devguide-pt
title: 'Translating the rOpenSci Dev Guide into Portuguese: Collaboration, Community, Challenges, and Impact'
author: 
  - Francesca Belem Lopes Palmeira
  - Beatriz Milz
  - Ariana Moura Cabral  
  - Yanina Bellini Saibene
  - Daniel Vartanian
  - Pedro Faria
date: '2025-11-25'

> editor: Steffi LaZerte

Reviewer 3: ```suggestion
featured: true
editor: Steffi LaZerte
```
